store/postgres_migration.go

```diff
@@ -121,7 +123,7 @@ func psqlMigrate(conn *pgxpool.Pool, tenant string, schema *bubblySchema, migr m
 func alterColumnStatement(provider config.StoreProviderType, tenant string, info tableInfo, columnType interface{}) ([]string, error) {
 	t, ok := (columnType).(cty.Type)
 	if !ok {
-		return nil, fmt.Errorf("not able to parse value to cty.Type: %s", columnType)
+		return nil, fmt.Errorf("cannot assign type %s to cty.Type", reflect.TypeOf(columnType).String())
```

Comment:
❓ The canonical logging format would be `"cannot assign type to core.TableJoin: %s"`, no?